experimental/Schemes/IdealSheaves.jl

```diff
@@ -335,3 +337,89 @@ function is_prime(I::IdealSheaf)
   return all(U->is_prime(I(U)), basic_patches(default_covering(space(I))))
 end
 
+function _minimal_power_such_that(I::Ideal, P::PropertyType) where {PropertyType}
+  whole_ring = ideal(base_ring(I), [one(base_ring(I))])
+  P(whole_ring) && return (0, whole_ring)
+  P(I) && return (1, I)
+  I_powers = [(1,I)]
+
+  while !P(last(I_powers)[2])
+    push!(I_powers, (last(I_powers)[1]*2, last(I_powers)[2]^2))
```

Comment:
Doubling the power with each iteration could be dangerous if 
I`^16` is just too difficult but `I^9` does the trick.

Reply:
Usually, I would expect small orders of vanishing and hence `I, I^2, I^3,...` should suffice?

Reply:
This was following a hint by @fieker from a long time ago in another context. I see your point, but would vote for first keeping an eye on this in practice. We do not need to compute a Groebner basis of the power, but 'only' a saturation. I'm not sure whether that makes things easier, but it could be possible. 

If it turns out that this is where stuff gets stuck, we can switch. But in general, this approach saves us a lot of time since it reduces `n` computations to `2*log(n)` computations. Maybe, we have to introduce another heuristic at some point, depending on the complexity of the input, to go incrementally from some power onwards. Let's see.